slash/react/src/Form/Text/__tests__/Text.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import { Text } from "../Text";
+
+describe("Text", () => {
+  it("should have value", () => {
+    // Act
+    render(<Text value="Hello World" />);
+
+    // Assert
+    expect(screen.getByRole("textbox")).toHaveValue("Hello World");
+  });
+
+  it("should have default class", () => {
+    // Act
+    render(<Text value="Hello World" />);
+
+    // Assert
+    expect(screen.getByRole("textbox")).toHaveClass("af-form__input-text");
+  });
```

Comment:
Peut-être tester les 2 dans le même test afin que les tests soient plus rapides
Qu'en penses-tu ?

Reply:
done and resolve merci de ton aide 💓